package.json

```diff
@@ -48,17 +48,18 @@
   },
   "homepage": "https://github.com/ng-bootstrap/ng-bootstrap#readme",
   "dependencies": {
-    "@angular/core": "4.0.3",
-    "@angular/common": "4.0.3",
-    "@angular/forms": "4.0.3"
+    "@angular/animations": "^4.3.5",
```

Comment:
This would be a breaking change for ng-bootstrap (bumping up minimal version of the dependency). Do you need `4.3.5`, specifically? What would be the earliest version of Angular that would work here?

Also, please don't use `^`.